core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -43,31 +44,45 @@ <h1 class="stat-value"><[dashboardStats.total_plays]></h1>
           <p class="stat-description">Open feedback</p>
           <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </div>
+        <div class="total-open-feedback stats-card">
+          <p class="stat-description">Subscribers</p>
+          <h1 class="stat-value"><[subscribersList.length]></h1>
+        </div>
       </md-card>
     </div>
 
-    <div ng-if="explorationsList.length > 0 || collectionsList.length > 0">
+    <div ng-if="explorationsList.length > 0 || collectionsList.length > 0 || subscribersList.length > 0">
```

Comment:
For this situation, I have modified the creator dashboard to look like this - 
![image](https://cloud.githubusercontent.com/assets/13691317/22968382/192caf72-f362-11e6-80d1-b330d520d15c.png)

Would this do?